src/org/isf/hospital/service/HospitalIoOperationRepository.java

```diff
@@ -5,10 +5,12 @@
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface HospitalIoOperationRepository extends JpaRepository<Hospital, String> {
 
-    @Query(value = "SELECT HOS_CURR_COD FROM HOSPITAL", nativeQuery= true)
-    String findHospitalCurrent();
+   @Query(value = "select h.currencyCod from Hospital h")
+	 List<String> findAllHospitalCurrencyCode();
```

Comment:
indentation

Reply:
Do we have any identation standard in project? Sometimes I see one tab sometimes two and sometimes 4 or spaces or even 3 ;) Anyway I will change to 4 spaces as it's more readable on github

Reply:
The actual code has indeed this problem of different styles, with #50 the author "added a .editorconfig file, so all editorconfig-aware editors will use tabs from now on."

Reply:
Ok I corrected it to tabs